finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java

```diff
@@ -41,27 +41,40 @@
     @GET
     @Path("/{hostname}")
     @Produces(MediaType.APPLICATION_JSON)
+    // tag::APIResponses[]
+    // tag::host-property[]
     @APIResponses(
         value = {
+            // tag::APIResponse[]
             @APIResponse(
                 responseCode = "404", 
                 description = "Missing description",
                 content = @Content(mediaType = "text/plain")),
+            // end::APIResponse[]
             @APIResponse(
                 responseCode = "200",
                 description = "JVM system properties of a particular host.",
+                // tag::Content[]
                 content = @Content(mediaType = "application/json",
+                // end::Content[]
+                // tag::Schmea[]
```

Comment:
Schema hotspot isn't working because `// tag::Schmea[]` and  `// end::Schmea[]` is spelled wrong.